testsuites/syncgateway/functional/tests/test_cbs_collections.py

```diff
@@ -635,6 +632,31 @@ def get_collection_stats(db, scope, collection, stats):
         return str(k)
 
 
+def verify_stats_retrieval(stats, auth, db, scope, collection):
+    """
+    Helper function for test_collection_stats
+    Verify stats can be procured as RBAC roles sgw_dev_ops, external_stats_reader and full admin
+    Verify correct stats are included, with correct initial values
+    """
+    collection_stats_keys = ["num_doc_writes", "num_doc_reads", "doc_writes_bytes", "doc_reads_bytes", "import_count", "sync_function_time",
+                             "sync_function_count", "sync_function_reject_access_count", "sync_function_reject_count", "sync_function_exception_count"]
+
+    try:
+        global_stats = stats["syncgateway"]["global"]
+    except KeyError:
+        assert False, f"Could not retrieve global stats via _expvar endpoint using RBAC user {auth[0]}"
+    else:
+        assert len(global_stats) != 0, f"Global stats are not reported correctly \n + {str(global_stats)}"
+
+    collection_stats = get_collection_stats(db, scope, collection, stats)
+
+    for key in collection_stats_keys:
+        try:
+            assert collection_stats[key] == 0, f"{key} statistic should be 0, got {collection_stats[key]}"
```

Comment:
This particular function is being run at the start of the test, so the stats should always be 0 for a new run of the test because we use a new set of buckets/dbs for each test.